src/model/chart-model.ts

```diff
@@ -80,6 +80,11 @@ export interface ChartOptions {
 	width: number;
 	/** Height of the chart */
 	height: number;
+	/**
+	 * Use observer to resize chart to its parent client area
+	 * Calling code is responsible for providing pilyfill if native implementation is not avaibable
+	 * */
```

Comment:
```suggestion
	 * Use ResizeObserver to handle container's resize to resize a chart automatically.
	 * Note that the library WON'T ADD own polyfill for `ResizeObserver` if it doesn't exist in the global scope. So you have to be sure that you enable this option only if a browser supports it or you provide a polyfill instead. Otherwise, a warn message will be printed and no handling resize will be applied.
	 **/
```

Maybe something like that?

Reply:
Probably we need to update docs as well accordingly.